### ISSUE TITLE
Have it available in JS only without the need to output in CSS?

### ISSUE THREAD

**Issue by binarykitchen** [mtojo/rollup-plugin-stylus-css-modules#5]
Can the compiled stylus file (raw CSS) land in the same JS file rollup is piping into?

If you can do

```
import styles from './styles.styl';
const container = `<div class="${styles.container}">...</div>`;
```

why the need for a CSS file the JS code is never going to require at all?

**Comment by binarykitchen** [comment]
well, to be exact, if I remove the output option, then it complains like

```
bundling...
🚨   Unexpected character '@'
src/styles/main.styl.css (1:0)
1: @-moz-keyframes _src_styles_main__blink {
   ^
2:   0% {
3:     opacity: 0.9;
```

**Comment by mtojo** [comment]
Since v1.3.0, if the `output` option is removed, the JS code is converted as follows:

Before:

```js
import styles from './styles.styl';
```

After:

```js
import styles from './styles.css';
```

This conversion was implemented to work with other CSS plugins.

-----

With [rollup-plugin-css-only](https://github.com/thgh/rollup-plugin-css-only) you can write like this:

```js
import stylusCssModules from 'rollup-plugin-stylus-css-modules';
import css from 'rollup-plugin-css-only';

export default {
  entry: 'path/to.js',
  plugins: [
    stylusCssModules(),
    css({output: false})
  ]
};
```

**Comment by mtojo** [comment]
I just released v1.4.0.

Try the following option:

```js
import stylusCssModules from 'rollup-plugin-stylus-css-modules';

export default {
  entry: 'path/to.js',
  plugins: [
    stylusCssModules({output: false})
  ]
};
```

**Comment by binarykitchen** [comment]
yeah, that worked. thanks man
